src/sentry/static/sentry/app/views/releases/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {RouteComponentProps} from 'react-router/lib/Router';
+
+type RouteParams = {
+  orgId: string;
+};
+
+type Props = RouteComponentProps<RouteParams, {}>;
+
+class ReleasesContainer extends React.Component<Props> {
+  render() {
+    const {children} = this.props;
+
+    return children;
```

Comment:
Why do you need this component if it doesn't really do anything. It is so you have a place to attach nested routes to?

Reply:
Yes, I wanted to have the exact same structure in v1 and v2